go.mod

```diff
@@ -100,6 +100,8 @@ require (
 	pgregory.net/rapid v0.4.7
 )
 
+require golang.org/x/tools v0.1.11
```

Comment:
you may want to move it to the block above.  wondering why did it decide to put it here....

Reply:
updated in https://github.com/onflow/flow-go/pull/3087/commits/635fbd32ef269e53491cecce5c373fa809f5a83e

I didn't want to move it manually after running `make fix-lint` which put it by itself. I moved it to the block above manually and re-ran `go mod tidy`